web/src/app/schedules/ScheduleShiftList.js

```diff
@@ -98,6 +99,95 @@ function ScheduleShiftList({ scheduleID }) {
     },
   })
 
+  function getShiftDetails(s, day) {
+    const tzAbbr = DateTime.local({ zone: zone }).toFormat('ZZZZ')
+    const localTzAbbr = DateTime.local({ zone: 'local' }).toFormat('ZZZZ')
+
+    let shiftDetails = ''
+    const startTime = s.start.toLocaleString({
+      hour: 'numeric',
+      minute: 'numeric',
+    })
```

Comment:
you could probably re-use the hour/minute code so you don't have to write that out each time here

```
const locale = {
  hour: 'numeric',
  minute: 'numeric',
}

const startTime = s.start.toLocaleString(locale)
const endTime = s.end.toLocaleString(locale)
const localStartTime = DateTime.fromISO(s.start, { zone: 'local' }).toLocaleString(locale)
const localEndTime = DateTime.fromISO(s.end, { zone: 'local' }).toLocaleString(locale)